apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -134,6 +147,19 @@ func (api *UserManagerAPI) RemoveUser(entities params.Entities) (params.ErrorRes
 		}
 		deletions.Results[i].Error = nil
 	}
+
+	// Security Event Logging: This log statement is required to comply with Canonical's SSDLC Security Event Logging policy.
+	// For details, see:
+	// https://library.canonical.com/corporate-policies/information-security-policies/ssdlc/ssdlc---security-event-logging
+	securitylog.LogUser(
+		securitylog.UserSecurityEvent{
+			Actor:  api.apiUser.Id(),
+			Target: fmt.Sprintf("%d users", len(entities.Entities)),
+			Action: "deleted",
+			Access: string(permission.NoAccess),
+		},
+	)
```

Comment:
This is incorrect (according to the spec). You need to dispatch this event for each user.